### PR TITLE
Fixes and renames, comments, and so on

### PR DESCRIPTION
**Pull request by eduser25**
Number of fixes, clarifications, name changes or proper addressing on a number of things.

### CODE REVIEW

File: tests/e2e/common.go [hunk 1]
```diff
@@ -165,8 +171,8 @@ type InstallOSMOpts struct {
 	vaultRole     string
 }
 
-// GetTestInstallOpts returns Install opts based on test flags
-func (td *OsmTestData) GetTestInstallOpts() InstallOSMOpts {
+// GetOSMInstallOpts it's a provided helper to initialize install options for OSM
```
Comment (nojnhuh) [inline tests/e2e/common.go]:
```suggestion
// GetOSMInstallOpts initializes install options for OSM
```

File: tests/e2e/common_apps.go [hunk 2]
```diff
@@ -150,7 +150,7 @@ func (td *OsmTestData) SimplePodApp(def SimplePodAppDef) (corev1.ServiceAccount,
 	return serviceAccountDefinition, podDefinition, serviceDefinition
 }
 
-// SimpleDeploymentAppDef defines some parametrization to create a deployment-based application from template
+// SimpleDeploymentAppDef defines some parametrization to create a pod-based application from template
```
Comment (nojnhuh) [inline tests/e2e/common_apps.go]: Did you mean to change this? Deployment makes more sense to me here.

File: tests/e2e/common_traffic.go [hunk 3]
```diff
@@ -12,20 +12,23 @@ import (
 )
 
 const (
-	// StatusCodeWord is a hardcoded word to use on curl command, to print and parse REST Status code
+	// StatusCodeWord is word identifier to use on curl command, to print and parse REST Status code
```
Comment (nojnhuh) [inline tests/e2e/common_traffic.go]:
```suggestion
	// StatusCodeWord is an identifier used on curl commands to print and parse REST Status codes
```

**Review by nojnhuh — APPROVED** [review]

File: tests/e2e/common_apps.go (hunk 2 above)
Comment (eduser25) [reply tests/e2e/common_apps.go]: ouch
```suggestion
// SimpleDeploymentAppDef defines some parametrization to create a deployment-based application from template
```

File: tests/e2e/e2e_deployment_client_server_test.go [hunk 4]
```diff
@@ -29,18 +29,17 @@ var _ = Describe("Test N Clients -> 1 server", func() {
 			sourceNamespaces = append(sourceNamespaces, fmt.Sprintf("%s%d", sourceAppBaseName, i))
 		}
 
-		It("Tests HTTP traffic for a simple client-server pod deployment", func() {
-			// For Cleanup only
-			td.Namespaces["server"] = true
+		It("Tests HTTP traffic for from multiple client deployments to a server deployment", func() {
```
Comment (eduser25) [inline tests/e2e/e2e_deployment_client_server_test.go]:
```suggestion
		It("Tests HTTP traffic from multiple client deployments to a server deployment", func() {
```

**Pull request merged**